docs/Analyzer Configuration.md

```diff
@@ -46,6 +46,8 @@ End users can enable `.editorconfig` based configuration for individual document
 
 Note that this additional file based approach is also supported on VS2019 16.3 and later releases for backwards compatibility.
 
+**The additional file based approach is no longer supported starting in Microsoft.CodeAnalysis.NetAnalyzers v(TODO). It will be implicitly discovered (if the file is in the project's directory or any ancestor directory), or it should be converted into a 'globalconfig'. See [Configuration files for code analysis rules](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/configuration-files).**
```

Comment:
@mavasani What version should I write in place of TODO? We likely want to include two versions, one for 5.x and one for 6.x?

Reply:
5.0.4 or later. Which reminds me that we should release the 5.0.4 NuGet package from main branch soon as we have accumulated quite a few bug fixes.

Reply:
```suggestion
**The additional file based approach is no longer supported starting in Microsoft.CodeAnalysis.NetAnalyzers v5.0.4. It will be implicitly discovered (if the file is in the project's directory or any ancestor directory), or it should be converted into a 'globalconfig'. See [Configuration files for code analysis rules](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/configuration-files).**
```